frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -209,7 +217,7 @@ const UploadPatients = () => {
     return (
       <div>
         {errorMessage.body && (
-          <div className="usa-alert usa-alert--error maxw-560">
+          <div className="usa-alert usa-alert--error maxw-560" tabIndex={-1}>
```

Comment:
Why is the tabIndex -1 necessary?

Reply:
`tabIndex` was added to make the div focusable and then the -1 is to only allow it be focused via `.focus()`.

Reply:
the focus should be place in the file input not the alert element and I think this alert is missing the role="alert"

Reply:
The ticket says `Ensure the error message gains focus once it appears on the page.`. We can talk to @emyl3 to confirm with them when we all get back on Tuesday. 
I can add role=alert on that one though.

Reply:
yeah, makes sense... I might be wrong